src/EPR.PRN.Backend.Obligation.UnitTests/Services/ObligationCalculatorServiceTests.cs

```diff
@@ -60,4 +56,14 @@ public async Task GetObligationCalculationById_ReturnsNull_WhenNoDataFound()
 
         result.Should().BeNull();
     }
+
+    [TestMethod]
+    public async Task ProcessApprovedPomData_ShouldBeCalled()
```

Comment:
Can we have a more descriptive name of what this unit test is doing in the name please. Similar to the other tests

Reply:
Updated